### DISCUSSION TITLE
Colour coding

### DISCUSSION THREAD

**Discussion by dna598** [helgoboss/helgobox#925] [Ideas]
Would it be possible to add some type of colour coding to the mappings? To be able to mark a bunch of mappings with different colours in order to find them easily.
Thank you.

**Comment by helgoboss** [comment]
Generally possible, yes. Mostly a question of where to put the color (both in mapping row and mapping panel) and how to transform it automatically, so that the GUI still looks acceptable. Ideas?

Here are the panels as they look now:

<img width="864" alt="image" src="https://github.com/helgoboss/realearn/assets/429285/30a41853-ced3-4c3c-89f4-5fae4b5e9a73">

<img width="833" alt="image" src="https://github.com/helgoboss/realearn/assets/429285/95ef49d0-d370-4e49-949c-25eeb6946a4d">

**Comment by dna598** [comment]
i dunno, maybe something like this?


![image](https://github.com/helgoboss/realearn/assets/169094695/24a39d4b-5d41-471e-a2f5-79cfa9290ad7)

![image](https://github.com/helgoboss/realearn/assets/169094695/abf0e1d1-2f0d-4e1f-9b8a-6620fb287f26)

**Comment by dna598** [comment]
Hi Boss, any progress with this? 

Differentiation by colour would really help.

**Comment by helgoboss** [comment]
No progress.
